qa/tasks/cephfs/test_journal_repair.py

```diff
@@ -403,3 +404,20 @@ def test_journal_smoke(self):
             "timeout": "1h"
         })
 
+    def test_import_from_empty_file(self):
+        """
+        That the 'journal import' recognizes empty file read and errors out.
+        """
+        fname = tempfile.NamedTemporaryFile(delete=False).name
+        self.mount_a.run_shell(["sudo", "touch", fname])
+        self.fs.fail()
+        import_out = self.fs.journal_tool(["journal", "import", fname], 0)
+        self.fs.set_joinable()
+        self.fs.wait_for_daemons()
+        try:
+            if import_out.endswith("done."):
```

Comment:
oh, isn't there a cleaner way to check this?

Reply:
... and there should be a test case to validate that journal import throws -EINVAL when importing from an invalid journal dump.

Reply:
I was wondering how to create an invalid dump file without manual editing? `journal export` doesn't create an invalid dump file or partial file.

Reply:
export the journal and then mangle first few bytes where the header would reside.

Reply:
It would be easier just by writing out an invalid header to a new file. That way, we know that it's an invalid dump file.

Reply:
Whatever works. My point is we need a test case.

Reply:
@joscollin I assume you are going to update this with a test case that validates importing from a corrupt journal.

Reply:
Yes, it's in progress.
Will update it soon.